app/scripts/controllers/transactions/index.js

```diff
@@ -2510,6 +2511,7 @@ export default class TransactionController extends EventEmitter {
       security_alert_reason:
         securityAlertResponse?.reason ?? BlockaidReason.notApplicable,
       ///: END:ONLY_INCLUDE_IN
+      gas_estimation_failed: Boolean(simulationFails),
```

Comment:
> We should also add a metrics event to track clicks of the "I will try anyways" button that is rendered if that failure happens:

I asked this during planning and @bschorchit said we don't need to as it's implicit in the fact that the transaction was submitted anyways. But I think an edge case will be when the user clicks the link and didn't submit the transaction. Is this something we want to track? @bschorchit 

> Question for the team: when the gas estimation fails, do we receive more information on it (e.g. a reason or error message)? If yes, it would be interesting to capture it as well within metrics.

No we don't receive any more information.

Reply:
> I asked this during planning and @bschorchit said we don't need to as it's implicit in the fact that the transaction was submitted anyways. But I think an edge case will be when the user clicks the link and didn't submit the transaction. Is this something we want to track?

No, we don't need to add metric for this edge case for now.

Reply:
gotcha and thanks for the clarification with the remaining comments @segun! maybe we should update the issue to reflect this as well. not exactly sure how to update it myself

Reply:
thanks for the clarification with the remaining comments! might be helpful to update the issue ticket with this information for visibility in case this comment gets hidden in the list of updates or by resolving this conversation